weasyl/followuser.py

```diff
@@ -152,6 +149,7 @@ def insert(userid, otherid):
         'ON CONFLICT DO NOTHING',
         user=userid, other=otherid, settings=WatchSettings.from_code(d.get_config(userid)).to_code())
 
+    from weasyl import welcome
```

Comment:
PEP 8 seems to apply here, and at L165? I know it technically _works_, but PEP seems to say to keep it up top?

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants

https://www.python.org/dev/peps/pep-0008/#imports